lang/en/local_aspiredu.php

```diff
@@ -56,3 +56,6 @@
 $string['privacy:metadata:aspiredu_provider:lastname'] = 'The last name of the user.';
 $string['privacy:metadata:aspiredu_provider:userid'] = 'The ID of the user.';
 $string['privacy:metadata:aspiredu_provider:username'] = 'The username of the user.';
+// New for lti 1.3
+$string['instance'] = 'Tool instance id';
```

Comment:
What is a typical value for this @xpazv?

Reply:
We need to support four different types of product integrations.

- Dropout Detective
- Grade Guardian
- Instructor Insight
- Retention Radar

This user experience of this flow isn't ideal. If they need to configure the external tools elsewhere (http://134.209.160.50/moodlegit/admin/settings.php?section=modsettinglti), why are we needing them to select something here? It should be possible to look up that tool where this parameter is being used.

Reply:
As it is implemented, right now, the user has to set up the external tool, at site level, and then, will have a block, available in dashboard or course context, where we show the user the access to the product integrations (currently DD and ii)

Reply:
I check with Kim. We're sticking with only DD and II. The other products are being tabled for now.